config/environments/development.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # In the development environment your application's code is reloaded on
-  # every request.  This slows down response time but is perfect for development
-  # since you don't have to restart the webserver when you make code changes.
+  # In the development environment your application's code is reloaded any time
+  # it changes. This slows down response time but is perfect for development
+  # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
+  config.action_controller.enable_fragment_cache_logging = true
```

Comment:
This seems like a dup of L20?

Can't spot it here:
http://railsdiff.org/5.2.4.4/6.1.1

Reply:
hmmm. Maybe a copy and paste error.